app/src/main/java/com/framgia/music_49/screen/audio/GenreAudioApdater.java

```diff
@@ -0,0 +1,69 @@
+package com.framgia.music_49.screen.audio;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import com.bumptech.glide.Glide;
+import com.framgia.music_49.data.model.Genre;
+import com.framgia.music_49.utils.ItemClickListener;
+import com.framgia_music_49.R;
+import java.util.List;
+
+public class GenreAudioApdater extends RecyclerView.Adapter<GenreAudioApdater.ItemHolder> {
+    private Context mContext;
+    private List<Genre> mGenres;
+    private ItemClickListener mItemClickListener;
+
+    public GenreAudioApdater(Context context, List<Genre> genres,
+            ItemClickListener itemClickListener) {
+        mContext = context;
+        mGenres = genres;
+        mItemClickListener = itemClickListener;
+    }
+
+    @NonNull
+    @Override
+    public ItemHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
+        View itemView;
+        itemView = LayoutInflater.from(viewGroup.getContext())
+                .inflate(R.layout.item_view_audio_genre, viewGroup, false);
+        return new ItemHolder(itemView, mItemClickListener);
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull ItemHolder viewHolder, int position) {
+        viewHolder.binData(mGenres.get(position));
+    }
+
+    @Override
+    public int getItemCount() {
+        return mGenres.size();
+    }
+
+    public class ItemHolder extends RecyclerView.ViewHolder implements View.OnClickListener {
```

Comment:
https://stackoverflow.com/questions/31302341/what-difference-between-static-and-non-static-viewholder-in-recyclerview-adapter
tìm hiểu tại sao lại để ViewHolder là static